packages/eas-json/schema/eas.schema.json

```diff
@@ -602,6 +602,11 @@
           "type": "string",
           "description": "The Bundle identifier that will be used when accessing submit credentials managed by Expo, it does not have any effect if you are using local credentials. In most cases this value will be autodetected, but if you have multiple Xcode schemes and targets, this value might be necessary."
         },
+        "groups": {
+          "type": "array",
+          "description": "Testing groups to send the build to.",
```

Comment:
I think it would be good to write more about this. What "groups", where can you define them etc.